Explain local testing config in README. (#69)
w3c/vc-data-model-2.0-test-suite: README.md

npm test
```

### Testing Locally

To test a single implementation or endpoint running locally, you can
copy `localConfig.example.cjs` to `localConfig.cjs`
in the root directory of the test suite.

```bash
cp localConfig.example.cjs localConfig.cjs
```

This file must be a CommonJS module that exports an object containing a
`settings` object (for configuring the test suite code itself) and an
`implementations` array (for configuring the implementation(s) to test against).

The format of the object contained in the `implementations` array is
identical to the one defined in

> [VC Test Suite Implementations](https://github.com/w3c/vc-test-suite-implementations?tab=readme-ov-file#usage)).

Reviewer: ```suggestion
[the **_Testing locally_** section of VC Test Suite Implementations](https://github.com/w3c/vc-test-suite-implementations?tab=readme-ov-file#testing-locally)).
```

> The `implementations` array may contain more than one implementation object, to
> test multiple implementations in one run.

Reviewer: ```suggestion
The `implementations` array may contain more than one implementation object,
enabling you to test multiple implementations in one run.
```

```js

> // localConfig.cjs defining local implementations

Reviewer: ```suggestion
// localConfig.cjs defines local implementations
```

> // you can specify a BASE_URL before running the tests such as:

Reviewer: ```suggestion
// Before running the tests, you can specify a BASE_URL, such as
```
